LLMonFHIR/FHIR Views/FHIRResourcesInstructionsView.swift

```diff
@@ -31,20 +31,17 @@ struct FHIRResourcesInstructionsView: View {
             VStack(alignment: .center) {
                 HStack {
                     Spacer()
-                    Button(
-                        action: {
-                            withAnimation {
-                                onboardingInstructions = false
-                            }
-                        },
-                        label: {
-                            Image(systemName: "xmark")
-                                .accessibilityLabel(Text("DISMISS_ONBOARDING_HINT"))
-                                .foregroundColor(.secondary)
-                        }
-                    )
+                    if #available(iOS 26.0, *) {
+                        dismissButton
+                        #if swift(>=6.2)
+                            .buttonStyle(.glass)
+                        #endif
+                    } else {
+                        dismissButton
+                    }
```

Comment:
@lukaskollmer That's what I was referring to in our meeting today. What do you think about adding some modifiers to SpeziViews that abstract this?
E.g., an
```swift
.apple26os { view in
    .buttonStyle(.glass)
}
```
modifier?

Reply:
Yes, that's unfortunately needed to identify if we run with Xcode 26. Would actually be nicer to identify the currently installed toolchain or something here to make it rather dependent on the iOS version but I am not aware that this is easily doable.